lib/api.js

```diff
@@ -36,6 +36,19 @@ function merge (a, b, mapper) {
   return a
 }
 
+/**
+ * @param {Record<string, any>} obj
+ * @param {{name?: string}} plugin
+ */
+function pluckOpts (obj, plugin) {
+  if (plugin.name) {
+    const camelCaseName = /** @type {string} */ (u.toCamelCase(plugin.name))
```

Comment:
:fire: this needs mentioning in readme briefly. You said conifg will be at config[pluginName] maybe mention pluginName is camelCase version of plugin name